tests/test_models.py

```diff
@@ -782,6 +854,34 @@ def test_delete_audit_action_audit_deletes_and_creates_audit_events(self):
             event.delta,
         )
 
+    def test_delete_audit_action_audit_rolls_back_if_make_audit_event_fails(self):  # noqa: E501
+        ModelWithAuditingManager.objects.create(id=0, value="initial")
+        queryset = ModelWithAuditingManager.objects.all()
+
+        with (patch('field_audit.models.AuditEvent.make_audit_event',
+                    side_effect=MakeAuditEventException()),
+              self.assertRaises(MakeAuditEventException)):
+            queryset.delete(audit_action=AuditAction.AUDIT)
+
+        instance = ModelWithAuditingManager.objects.get(id=0)
+        self.assertIsNotNone(instance)
```

Comment:
Curious why this is implemented differently from the similar check (with `try`/`except`/`self.fail`) in the next test?

Also, `.get(id=0)` will fail with `DoesNotExist` and `self.assertIsNotNone` will never be called if the object is deleted, right?

Reply:
Huh that is a great point. No clue what I was thinking, and am inclined to modify this test to have a similar check to the `try`/`except`/`self.fail` check.